src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, AttributesDrawing {
+    private final String color;
+
+    Figure(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    @Override
+    public double getArea() {
+        return 0;
+    }
```

Comment:
The 'getArea' method should not be implemented here if it's intended to be specific to each figure. Remove the implementation from the abstract class and ensure that each subclass provides its own implementation.